ccdproc/image_collection.py

```diff
@@ -109,6 +109,11 @@ def __init__(self, location=None, keywords=None, info_file=None,
         # is always *some* value.
         self._all_keywords = False
 
+        if extension is None:
```

Comment:
I'm not sure but you could also set the default value to `0`. That should have the same effect as that `if ... else ...`, right?